crates/primitives/src/compression/mod.rs

```diff
@@ -38,6 +39,38 @@ thread_local! {
         ));
 }
 
+/// Tx compressor creator
```

Comment:
please update these comments to be in the same style as the rest of the codebase, i.e. "Creates a [`Compressor`] for transactions" or something like that

Reply:
Done